src/pages/en/core-concepts/astro-pages.md

```diff
@@ -5,17 +5,30 @@ description: An introduction to Astro pages
 i18nReady: true
 ---
 
-**Pages** are a special type of [Astro component](/en/core-concepts/astro-components/) that live in the `src/pages/` subdirectory. They are responsible for handling routing, data loading, and overall page layout for every HTML page in your website.
+**Pages** are special files that live in the `src/pages/` subdirectory. They are responsible for handling routing, data loading, and overall page layout for every HTML page in your website.
```

Comment:
Question: are the files “special”? Or is it that `src/pages/` is special? I wonder if “special” here suggests you need to do something different to these files but actually putting them in `pages` was already all that was needed.

```suggestion
**Pages** are files that live in the `src/pages/` subdirectory of your Astro project. They are responsible for handling routing, data loading, and overall page layout for every HTML page in your website.
```

Reply:
Yep, `special` was a holdover from "special kind of Astro component", but even there it might've been unnecessary!